api/v1/notification_types.go

```diff
@@ -0,0 +1,79 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type NotificationRecipientSpec struct {
```

Comment:
I think we are missing properties here as well

Reply:
Why do we need it in here as we already have it in the notification spec?

Reply:
It should be in here instead of the notification spec, 

```yaml
to:
    url: smtp://system
    properties:
       FromName: ABC
```

vs which is unclear exactly what the properties apply to

```yaml
to:
    url: smtp://system
properties:
    FromName: ABC
```